### PR TITLE
chore: rdr2 referencess

### PR DESCRIPTION
**Pull request by maxwelbm**
https://reddead.fandom.com/pt-br/wiki/Gangue_Van_der_Linde

### CODE REVIEW

**Review by PatrickMenoti — DISMISSED** [review]

**Review by tiagokrebs — CHANGES_REQUESTED** [review]
A little bit picky but maybe we should observe the [CONTRIBUTING](https://github.com/aziontech/go-thoth/blob/main/CONTRIBUTING.md) here, specially the rule 3.
> Respect: Please ensure that the names suggested are respectful and inclusive. Avoid any words that might be considered discriminatory, inappropriate, or disrespectful in any form.

Terms like "Crook", "Thief", "Harlot" could be considered disrespectful or carry negative connotations.  
Maybe more terms besides these should not be used, please look at [this conversation](https://chat.openai.com/share/7482d1c9-b0b5-4d30-b89e-c20a7fdee022) with ChatGPT about it.

**Review by PatrickMenoti — DISMISSED** [review]

File: nouns.go [hunk 1]
```diff
@@ -154,5 +154,39 @@ func GetNouns() []string {
 		"Rohan",
 		"Mordor",
 		"Isengard",
+		"VanDerLinde",
+		"DutchLinde",
+		"HoseaMatthews",
+		"ArthurMorgan",
+		"JohnMarston",
+		"JavierEscuella",
+		"JavierEscuella",
```
Comment (tiagokrebs) [inline nouns.go]: Double noun here =D

File: nouns.go [hunk 2]
```diff
@@ -154,5 +154,39 @@ func GetNouns() []string {
...
+		"LennySummers",
+		"SadieAdler",
+		"KarenJones",
+		"TillyJackson",
+		"Mary-BethGaskill",
+		"Uncle",
+		"AbigailRoberts",
+		"SusanGrimshaw",
+		"SimonPearson",
+		"SimonPearson",
```
Comment (tiagokrebs) [inline nouns.go]: Double noun here too =D

File: nouns.go [hunk 3]
```diff
@@ -154,5 +154,39 @@ func GetNouns() []string {
...
+		"LeopoldStrauss",
+		"JosiahTrelawny",
+		"OrvilleSwanson",
+		"OrvilleSwanson",
```
Comment (tiagokrebs) [inline nouns.go]: Double noun here again =D

File: nouns.go [hunk 4]
```diff
@@ -154,5 +154,39 @@ func GetNouns() []string {
...
+		"KieranDuffy",
+		"MollyOShea",
+		"DaveyCallander",
+		"MacCallander",
+		"JennyKirk",
+		"Annabelle",
+		"BessieMatthews",
+		"BessieMatthews",
```
Comment (tiagokrebs) [inline nouns.go]: Last double noun =D

**Review by tiagokrebs — COMMENTED** [review]
Found some double nouns, also, what about this:
> Nouns: The nouns used for suggesting names should be simple, respectful, and related to technology or nerdy content. They should be a single word. Avoid using jargon or acronyms that might not be universally understood.

Some nouns do not have a single word, but many in Cammel case, this is a new pattern, maybe we should follow the guideline, or change it.

**Comment by tiagokrebs** [comment]
@maxwelbm if you're not willing to pursue this change anymore we can close this pull request.

**Comment by maxwelbm** [comment]
I'm going to add a test to check for duplicate adjectives or nouns, to avoid cases like this, I'll bring it up in a separate pr @tiagokrebs

**Review by tiagokrebs — APPROVED** [review]

**Review by PatrickMenoti — APPROVED** [review]
